packages/query/src/stacks/balance/index.ts

```diff
@@ -0,0 +1,11 @@
+import { createGetStacksAccountBalanceQueryOptions } from './account-balance.query';
+import { createStxCryptoAssetBalance } from './create-stx-crypto-asset-balance';
+import { createStxMoney } from './create-stx-money';
+import { useStxBalanceQuery } from './use-stx-balance-query.hooks';
+
+export {
+  createGetStacksAccountBalanceQueryOptions,
+  createStxCryptoAssetBalance,
+  createStxMoney,
+  useStxBalanceQuery,
+};
```

Comment:
Is this file necessary, why not just export at the root level like the rest of the exports?

Reply:
I will get rid of it when cleaning up 👍

Reply:
Removed